content/en/platform/corda/4.12/enterprise/platform-support-matrix.md

```diff
@@ -35,24 +35,21 @@ for your deployment needs.
 
 ### JDK support in development
 
-The following JDKs support Corda for development purposes. Corda does not currently support Java 9 or higher.
+The following JDKs support Corda for development purposes. Corda does not currently support Java 18 or higher.
 
 {{< table >}}
 
 |Supported JDKs|Latest supported version|
 |-----------------------------------|-----------|
-|[Oracle JDK](https://www.oracle.com/ie/java/technologies/downloads/)|8u381|
-|[Amazon Corretto 8](https://aws.amazon.com/corretto/)|8.252.09.1|
-|[Red Hat OpenJDK](https://developers.redhat.com/products/openjdk/overview/)|8u322|
-|[Zulu OpenJDK](https://www.azul.com/)|8u382|
+|[Zulu OpenJDK](https://www.azul.com/)|17.0.9|
```

Comment:
_Sorry_ Anna, 

|[Oracle JDK](https://www.oracle.com/ie/java/technologies/downloads/)|17.0.9|

needs to be added back in now